pkg/image/qemu.go

```diff
@@ -137,6 +141,33 @@ func convertToRaw(src, dest string, preallocate bool) error {
 	return nil
 }
 
+func getCacheMode(path string) (string, error) {
+	if os.Getenv(common.CacheMode) != common.CacheModeTryNone {
```

Comment:
Does this package look at environment variables anywhere else?  if not then maybe it wiould be better to have this as a function param?

Reply:
Yeah better to keep env checking in cmd/. Posted the change